src/components/picker/PickerDialog.ios.js

```diff
@@ -56,7 +56,7 @@ class PickerDialog extends BaseComponent {
     // TODO: should be taken from dialogProps but there's an issue with "babel-plugin-typescript-to-proptypes" plugin
     const {panDirection} = this.props;
     return (
-      <Dialog {...dialogProps} height={250} width="100%" migrate bottom animationConfig={{duration: 300}} panDirection={panDirection}>
+      <Dialog {...dialogProps} width="100%" migrate bottom animationConfig={{duration: 300}} panDirection={panDirection}>
```

Comment:
@mendyEdri 
Regarding 
> Unwanted extra space in the WheelPicker when it's under the safeArea.

Consider doing this 

```
      <Dialog {...dialogProps} width="100%" migrate bottom animationConfig={{duration: 300}} panDirection={panDirection}>
        <View flex bg-white>
          {this.renderHeader()}
          <View centerV flex>
            {this.renderPicker()}
          </View>
          <View useSafeArea/>
        </View>
      </Dialog>
```

I added `<View useSafeArea/>` under the wrapper View of the picker.